src/Answers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+
+const Answers = ({answers}) => {
+  let answersHTML = answers.map((answer, index) => {
+    return (
+      <p className="quiz-answer" key={index}>{answer.title}</p>
```

Comment:
I would maybe make each answer a `label` element with a `radio` button so that they are selectable.